cmake/init_remote_projects.cmake

```diff
@@ -0,0 +1,24 @@
+message(STATUS "Remote building selected. Skipping everything except IDE initialization...")
+if("${USE_IDE}" STREQUAL "eclipse")
+    message(WARNING "Remote Execution is not supported in Eclipse! Use VSCode or the Command line!")
+elseif("${USE_IDE}" STREQUAL "vscode")
```

Comment:
There are a few differences between docker and remote builds, so some templates are shared, some custom.